tests/model_fields_/array_index_migrations/0001_initial.py

```diff
@@ -30,6 +30,13 @@ class Migration(migrations.Migration):
                     "text",
                     django_mongodb_backend.fields.ArrayField(models.TextField(), db_index=True),
                 ),
+                ("char3", models.CharField(max_length=11, db_index=True)),
```

Comment:
Probably we don't need this test... it's for a PostgreSQL issue (https://github.com/django/django/commit/dad8434d6ff5da10959672726dc9b397296d380b), but at least you don't need to extend it in this PR.